Makefile

```diff
@@ -4,4 +4,4 @@ setup:
 	go get github.com/stretchr/testify
 
 test:
-	go test -v github.com/ashwanthkumar/go-gocd
+	go test -v github.com/pagero/go-gocd-ashwanth
```

Comment:
Can you please fix the package names across all the files and the README? I would love to merge the other changes.

Reply:
Oops, forgot that we opened this PR from our fork's master branch... will close this PR and open another one that only includes the API version bump.